lib/common/xml.c

```diff
@@ -1104,12 +1103,14 @@ pcmk__xml_needs_escape(const char *text, bool for_attr)
                 }
                 break;
             default:
-                if ((text[index] < 0x20) || (text[index] >= 0x7F)) {
+                if ((*text < 0x20) || (*text >= 0x7F)) {
```

Comment:
We should be able to use `isprint()` here

Reply:
That's handy